pyuvdata/uvdata/mir_parser.py

```diff
@@ -1761,7 +2108,13 @@ def _fix_acdata(self):
 
     def read(
         self,
-        filepath,
+        filepath=None,
+        *,
+        auto_defaults=True,
+        compass_soln=None,
+        make_v3_compliant=None,
```

Comment:
Should this also be a bool?

Reply:
So I typically use the `None` setting as a way to setup a more dynamic default behavior beyond the normal `True`/`False`. In this instances, I set this to true if the file version is determined to be < 3, otherwise set to False. I'm apt to keep this setting, but I realize that the `auto_defaults` keyword is kind of redundant and not actually necessary, so I'm going to remove that and keep the ascribed `None` behavior.